### PR TITLE
add bookstore recipe

### PR DESCRIPTION
**Pull request by willingc**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
thanks Carol!

File: recipes/bookstore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac8d72754a283ccccc6483d518d952bfdc5ba072a1fff2fbe736fa7e02c69d9c
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/bookstore/meta.yaml]: This can be `noarch`
```suggestion
  number: 0
  noarch: python
```

File: recipes/bookstore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - future
+    - ipython>=5.0
```
Comment (xhochy) [inline recipes/bookstore/meta.yaml]: Please leave a space between name and spec
```suggestion
    - ipython >=5.0
```

File: recipes/bookstore/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - notebook
+    - tornado>=5.1.1
```
Comment (xhochy) [inline recipes/bookstore/meta.yaml]:
```suggestion
    - tornado >=5.1.1
```

**Comment by xhochy** [comment]
Thanks @willingc

**Pull request merged**